graspologic/align/seeded_procrustes.py

```diff
@@ -0,0 +1,186 @@
+    graphs, which requires some simultaneous inference task and partial 1-1
+    matching between the vertices of the two graphs.
+
+    Attributes
+    ----------
+    Q_ : array, size (d, d)
+        Final orthogonal matrix, used to modify ``X``.
+
+    References
+    ----------
+
+    .. [1] https://en.wikipedia.org/wiki/Orthogonal_Procrustes_problem
+
+    .. [2] Peter H. Schonemann, "A generalized solution of the orthogonal
+           Procrustes problem", Psychometrica -- Vol. 31, No. 1, March, 1996.
+
+    .. [3] Agterberg, J., Tang, M., Priebe., C. E. (2020).
+        "Nonparametric Two-Sample Hypothesis Testing for Random Graphs with Negative and Repeated Eigenvalues"
+        arXiv:2012.09828
+
+    .. [4] Agterberg, J., Tang, M., Priebe., C. E. (2020).
+        "On Two Distinct Sources of Nonidentifiability in Latent Position Random Graph Models"
+        arXiv:2003.14250
+
+    Notes
+    -----
+    The goal of this process is to find a correspondence between
+    the vertices of two datasets as well as the orthogonal alignment
+    between them. If the two datasets are represented with
+    matrices :math:`X \in M_{n, d}` and :math:`Y \in M_{m, d}`,
+    then the correspondence is a matrix :math:`P \in M_{n, m}` that is
+    soft assignment matrix (that is, its rows sum to :math:`1/n`, and columns
+    sum to :math:`1/m`) and the orthogonal alignment is an
+    orthogonal matrix :math:`Q \in M_{d, d}`. An orthogonal
+    matrix is any matrix that satisfies :math:`Q^T Q = Q Q^T = I`.
+    The global objective function is :math:`|| X Q - P Y ||_F`.
+
+    Note that both :math:`X` and :math:`PY` are matrices in :math:`M_{n, d}`.
+    Thus, if one knew :math:`P`, it would be simple to obtain an estimate for
+    :math:`Q`, using the regular orthogonal procrustes. On the other hand, if
+    :math:`Q` was known, then :math:`XQ` and :math:`Y` could be thought of
+    distributions over a finite number of masses, each with weight :math:`1/n`
+    or :math:`1/m`, respectively. These distributions could be "matched" via
+    solving an optimal transport problem.
+
+    However, both :math:`Q` and :math:`P` are simultaneously unknown here, so
+    the algorithm performs a sequence of alternating steps, obtaining
+    iteratively improving estimates of :math:`Q` and :math:`P`, similarly to an
+    expectation-maximization (EM) procedure. It is not known whether this
+    procedure is formally an EM, but the analogy can be drawn as follows: after
+    obtaining an initial guess of of :math:`\hat{Q}_{0}`, obtaining an
+    assignment matrix :math:`\hat{P}_{i+1} | \hat{Q}_{i}` ("E-step") is done by
+    solving an optimal transport problem via Sinkhorn algorithm, whereas
+    obtaining an orthogonal alignment matrix :math:`Q_{i+1} | P_{i}`
+    is done via regular orthogonal procurstes. These steps are further simplified
+    by the use of a seeded matrix that contains rows of seeds that align a set of
+    indeses in ``X`` and ``Y``.
```

Comment:
I would replace all of this by saying that this method uses orthogonal procrustes on the data for which a 1-to-1 matching of elements is known, and then passes this as an initial guess to seedless procrustes. You dont need to copy all of this long-winded math/explanation. You can point to both of these classes for more information (some classes have a "see also" section).